### ISSUE TITLE
beautifier.min.js is not working

### ISSUE THREAD

**Issue by mittalalok** [beautify-web/js-beautify#1608]
# Description
 The minified version of beautifier.js is not working.
 When I am using the beautifier.min.js file in browser it throwing an error "Uncaught SyntaxError: Invalid regular expression: /[$@A-Z_a-zÂªÂµÂºÃ€-Ã–Ã˜-Ã¶Ã¸-ËË†-Ë" ....

The sample HTML file 
<!DOCTYPE html>
<html>
<body>
<div id="demo"></div>
<script src="https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.8.9/beautifier.min.js">
</script>
<script>
	document.getElementById("demo").innerHTML = "Hello JavaScript!";
</script>
</body>
</html>

**Comment by bitwiseman** [comment]
@mittalalok 
Interesting.  It looks like this is bug in cdnjs or in the way the project is being pulled into cdnjs.  

Compare this: 
https://raw.githubusercontent.com/beautify-web/js-beautify/v1.8.9/js/lib/beautifier.min.js
with the cdnjs version: 
https://cdnjs.cloudflare.com/ajax/libs/js-beautify/1.8.9/beautifier.min.js

In the regular expression in question:
```js
// github
e.identifier=new RegExp("[$@A-Z_a-zªµºÀ-ÖØ-öø-ˁˆ-ˑˠ-ˤˬˮͰ-ʹͶͷͺ-ͽΆ...")
// cdnjs
e.identifier=new RegExp("[$@A-Z_a-zÂªÂµÂºÃ€-Ã–Ã˜-Ã¶Ã¸-ËË...")
// cdn.rawgit.com
e.identifier=new RegExp("[$@A-Z_a-zªµºÀ-ÖØ-öø-ˁˆ-ˑˠ-ˤˬˮͰ-ʹͶͷͺ-ͽΆ...")
```

**Comment by bitwiseman** [comment]
Update: 
This is definitely a server configuration issue on cdnjs. 
See  https://github.com/cdnjs/cdnjs/issues/13191 .

**Comment by bitwiseman** [comment]
@mittalalok 
This should be fixed in 1.8.10-beta3.  Please give it a try.

**Comment by mittalalok** [comment]
@bitwiseman Thanks a lot, I will try it.
